src/tree_cmd.cpp

```diff
@@ -305,6 +333,7 @@ void GenerateTrees()
 
 	switch (_settings_game.game_creation.tree_placer) {
 		case TP_ORIGINAL: i = _settings_game.game_creation.landscape == LT_ARCTIC ? 15 : 6; break;
+		case TP_FOREST: /* FALL THROUGH */
```

Comment:
We have a proper macro for this now (`FALLTHROUGH`), to take advantage of some C++17 stuff if supported.